PacketDotNet/TcpFields.cs

```diff
@@ -29,14 +29,18 @@ public struct TcpFields
     {
 #pragma warning disable 1591
         // flag bitmasks
-        public readonly static int TCP_CWR_MASK = 0x0080;
-        public readonly static int TCP_ECN_MASK = 0x0040;
-        public readonly static int TCP_URG_MASK = 0x0020;
-        public readonly static int TCP_ACK_MASK = 0x0010;
-        public readonly static int TCP_PSH_MASK = 0x0008;
-        public readonly static int TCP_RST_MASK = 0x0004;
-        public readonly static int TCP_SYN_MASK = 0x0002;
-        public readonly static int TCP_FIN_MASK = 0x0001;
+        public readonly static int TCP_FLAGC_MASK = 0x0800;
```

Comment:
Why reverse the order of the masks here? It would be clearer if you added the new fields in the style of the existing masks, or separated out reversing the order of the masks into a separate commit, and then introduced the new fields. Doing both means I have to expend effort to distinguish between what you added and what you reordered.